google-cloud-spanner-executor/src/main/java/com/google/cloud/executor/spanner/CloudExecutorImpl.java

```diff
@@ -34,8 +37,14 @@ public class CloudExecutorImpl extends SpannerExecutorProxyGrpc.SpannerExecutorP
   // Executors to proxy.
   private final CloudClientExecutor clientExecutor;
 
-  public CloudExecutorImpl(boolean enableGrpcFaultInjector) {
+  // Ratio of operations to use multiplexed sessions. Only applies to supported operations
```

Comment:
```suggestion
  // Ratio of operations to use multiplexed sessions.
```

We don't need to detail out what operations are supported over here.